balancer/rls/balancer.go

```diff
@@ -139,13 +140,28 @@ type rlsBalancer struct {
 	// default child policy wrapper when a new picker is created. See
 	// sendNewPickerLocked() for details.
 	lastPicker *rlsPicker
+	// Set during UpdateClientConnState when pushing updates to child policies.
+	// Prevents state updates from child policies causing new pickers to be sent
+	// up the channel. Cleared after all child policies have processed the
+	// updates sent to them, after which a new picker is sent up the channel.
+	inhibitPickerUpdates bool
+
+	// Channel on which all updates are pushed. Processed in run().
+	updateCh *buffer.Unbounded
+}
+
+type resumePickerUpdates struct {
+	done chan struct{}
+}
 
-	// Channels on which updates are received or pushed.
-	ccUpdateCh               chan *balancer.ClientConnState
-	childPolicyStateUpdateCh *buffer.Unbounded // idAndState from child policy.
-	connectivityStateCh      chan struct{}     // signalled when control channel becomes READY again.
+// childPolicyIDAndState wraps a child policy id and its state update.
+type childPolicyIDAndState struct {
+	id    string
+	state balancer.State
 }
 
+type connectivityStateCh struct{}
```

Comment:
Can we find a better name?  `controlChannelReady`?

Reply:
Done.